omega2/omega2.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+import subprocess
+
+
+class Omega2(object):
+    def __init__(self):
+        led_name = self.get_led_name()
+        self.version = led_name.split(":")[0]
+        self.led_path = "/sys/class/leds/%s/" % led_name[:-1]
+        self.RGB_pins = {'R': '17', 'G': '16', 'B': '15'}
+
+        # https://docs.onion.io/omega2-docs/using-gpios.html#fast-gpio
+        self._gpio_tool = ['gpioctl', 'fast-gpio'][1]
+
+    def _shell(self, cmd_args, check_output=False):
+        if check_output:
+            return str(subprocess.check_output(cmd_args))
+        else:
+            return subprocess.call(cmd_args)
+
+    def get_led_name(self):
+        """Returns the internal name of the onboard LED"""
+        return self._shell(['uci', 'get', 'system.@led[0].sysfs'], True)
+
+    def led_control(self, state, delay_on=None, delay_off=None,
+                    message=None, morse_speed=None):
+        """
+        Pass a boolean argument to turn the Omega2 LED on and off.
+        Pass a string to trigger the LED mode.
+        Available modes: 'none', 'mmc0', 'timer', 'default-on', 'netdev',
+                         'transient', 'gpio', 'heartbeat', 'morse', 'oneshot'.
+            - When switching to the 'timer' mode, pass the *delay_on* and
+                *delay_off* arguments.
+            - When switching to the 'morse' mode, pass the *message* argument.
+        Details: https://docs.onion.io/omega2-docs/the-omega-led.html
+        """
+        if type(state) in {bool, int}:  # pylint: disable=C0123
```

Comment:
Maybe I can do it better, but it feels overcomplicated 
`if any({isinstance(state, x) for x in {bool, int}}):`